tests/suite/django.html

```diff
@@ -195,3 +195,32 @@ <h1>Welcome, {{ request.user }}</h1>
 {% customtag %}
     <div></div>
 {% endcustomtag %}
+
+<!-- Two-line custom tag -->
```

Comment:
Are these all valid formats?

Reply:
Do you mean valid according to this PR, or valid according to Qromponents?

Reply:
the answer should be yes either way 😄

Reply:
I meant, does this file contain only "valid" formats, according to djhtml?

Reply:
yeah, running the test suite unindents every line in that file, applies djhtml, and asserts equal with the original file. tests succeed with the examples added to that html file